src/app/meme-dialog/meme-dialog.component.ts

```diff
@@ -0,0 +1,46 @@
+import {Component, EventEmitter, Input, OnInit, Output} from '@angular/core';
+import {MatDialogModule} from '@angular/material';
+import {Utils} from '../utils';
+
+@Component({
+  selector: 'app-meme-dialog',
+  templateUrl: './meme-dialog.component.html',
+  providers: [MatDialogModule],
+  styleUrls: ['./meme-dialog.component.scss']
+})
+export class MemeDialogComponent implements OnInit {
+  @Input() username: string;
+  @Input() image: string;
+  @Input() totalVote: number;
```

Comment:
The parent should only pass the memeId and then you should use the MemeService to fetch the attributes.